packages/contracts/contracts/Unirep.sol

```diff
@@ -60,6 +60,7 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
     mapping(uint256 => IncrementalTreeData) public globalStateTree;
     // epoch => root => bool
     mapping(uint256 => mapping(uint256 => bool)) public globalStateTreeRoots;
+    mapping(uint256 => uint256) public epochTreeRoots;
```

Comment:
I still think we don't need this structure. We can just read the values using the existing function.

Reply:
[ca8e699](https://github.com/Unirep/Unirep/pull/126/commits/ca8e699b3fdab84da63f4aaf33543b86043b2dfc)